bindings/src/lib.rs

```diff
@@ -95,6 +98,29 @@ fn sign(entropy: String, message: String) -> String {
     STANDARD.encode(signature)
 }
 
+#[no_mangle]
+fn derive_alias(entropy: String, context: String) -> String {
+    let entropy_slice = match decode_entropy(&entropy) {
+        Ok(slice) => slice,
+        Err(err) => return err,
+    };
+
+    let secret = BandersnatchVrfVerifiable::new_secret(entropy_slice);
+
+    let context_slice = match decode_slice(&context) {
+        Ok(slice) => slice,
+        Err(err) => return err,
+    };
+
+    let alias = BandersnatchVrfVerifiable::alias_in_context(
+        &secret,
+        &context_slice
+    )
+    .unwrap();
```

Comment:
Let's use `.map_error(|e| error())?` instead of `.unwrap()` to prevent from crashing. Also, could you, please fixes unwraps in other function in the same way as well?